e2e/containers.e2e.spec.ts

```diff
@@ -0,0 +1,83 @@
+import path from 'path';
+import util from 'util';
+
+import { ElectronApplication, BrowserContext, _electron, Page } from 'playwright';
+import { test, expect } from '@playwright/test';
+import { createDefaultSettings, packageLogs, reportAsset, tool } from './utils/TestUtils';
+import { NavPage } from './pages/nav-page';
+
+let page: Page;
+
+/**
+ * Using test.describe.serial make the test execute step by step, as described on each `test()` order
+ * Playwright executes test in parallel by default and it will not work for our app backend loading process.
+ * */
+test.describe.serial('Containers Test', () => {
+  let electronApp: ElectronApplication;
+  let context: BrowserContext;
+
+  test.beforeAll(async() => {
+    createDefaultSettings();
+
+    electronApp = await _electron.launch({
+      args: [
+        path.join(__dirname, '../'),
+        '--disable-gpu',
+        '--whitelisted-ips=',
+        // See src/utils/commandLine.ts before changing the next item as the final option.
+        '--disable-dev-shm-usage',
+        '--no-modal-dialogs',
+      ],
+      env: {
+        ...process.env,
+        RD_LOGS_DIR: reportAsset(__filename, 'log'),
+      },
+    });
+    context = electronApp.context();
+
+    await context.tracing.start({ screenshots: true, snapshots: true });
+    page = await electronApp.firstWindow();
+  });
+
+  test.afterAll(async() => {
+    await context.tracing.stop({ path: reportAsset(__filename) });
+    await packageLogs(__filename);
+    await electronApp.close();
+  });
+
+  test('wait for the container engine to be ready', async() => {
+    const navPage = new NavPage(page);
+
+    await navPage.progressBecomesReady();
+    await expect(navPage.progressBar).toBeHidden();
+  });
+
+  test('should run uname -m on containers from different architectures', async() => {
+    // Are we running moby or containerd? Run tests with that engine first,
+    // and then switch to the other.
+    const settingsOutput = await tool('rdctl', 'list-settings');
+    const settings = JSON.parse(settingsOutput);
```

Comment:
```suggestion
    const settings: Settings = JSON.parse(settingsOutput);
```
(Just so we get type checking for the result.)

Do we even need the `settingsOutput` temporary?

Reply:
No, we don't the temporary. In answer to the first point, from https://docs.docker.com/desktop/multi-arch/ it looks like `linux/amd` covers the x86_64 platform (certainly the output for `uname -m` says it does).